### PR TITLE
CART-89 efa: Disable EFA provider

### PR DESCRIPTION
**Pull request by frostedcmos**
- Compile OFI with --disable-efa to disable EFA provider

Signed-off-by: Alexander Oganezov <alexander.a.oganezov@intel.com>

### CODE REVIEW

File: utils/rpms/daos.spec [hunk 1]
```diff
@@ -386,6 +386,9 @@ getent passwd daos_server >/dev/null || useradd -M daos_server
 %{_libdir}/*.a
 
 %changelog
+* Thu Oct 1 2020 Alexander Oganezov <alexander.a.oganezov@intel.com> 1.1.0-34
+- Set OFI to compile with --disable-efa flag to disable EFA provider
```
Comment (jolivier23) [inline utils/rpms/daos.spec:390]: don't think spec change is necessary here since the change doesn't affect the RPM build

Comment (brianjmurrell) [reply utils/rpms/daos.spec:390]: Agreed.

**Review by brianjmurrell — CHANGES_REQUESTED** [review]

**Review by jolivier23 — APPROVED** [review]
by reverting the spec change you avoided a conflict

**Comment by brianjmurrell** [comment]
> by reverting the spec change you avoided a conflict

And prevented a conflict for somebody else.  :-)

**Review by brianjmurrell — APPROVED** [review]

**Review by jolivier23 — APPROVED** [review]

**Pull request merged**